herd/mem.ml

```diff
@@ -44,7 +44,7 @@ module type S = sig
       loop_present : bool ;
      }
 
-  val glommed_event_structures : S.test -> result
+  val glommed_event_structures : S.test -> ((string * string) list) -> result
```

Comment:
External parenthesis of second tye argument can be removed
```suggestion
  val glommed_event_structures : S.test -> (string * string) list -> result
```

Reply:
good spot, will fix